webhook-bridge/svix-webhook-bridge.example3.yaml

```diff
@@ -0,0 +1,74 @@
+log_level: "debug"
+log_format: "json"
+
+# Optionally configure these settings to export OTEL data
+opentelemetry:
+  address: "http://localhost:1234"
+  service_name: "my-bridge"
+  sample_ratio: 0.5
+
+# Host/port to listen on when receiving webhooks
+receiver_http_addr: "0.0.0.0:5000"
+
+# Global svix settings.
+# A svix client is needed by each sender.
+# Each sender can override any of these fields by defining a `svix_client` section.
+global_svix_client:
+  token: "XXXXX"
+  options:
+    debug: true
+    server_url: http://example.com/
+
+# Webhook Senders configure an input and (optional) transformation.
+# Use transformations to conform the payload we get as input to the expected shape (app_id, message, etc).
+# When transformation is left unset, the payload is used as-is.
+senders:
```

Comment:
TBH, I think the example files should just have either senders or receivers. As in you'll have one file demonstrating one and another demonstrating the other. We probably don't want an example file that has both just for simplicity. We can mention in the docs that it's possible though.

At least for the start.

Reply:
I split things up in `.example5.senders.yaml` and `.example5.receivers.yaml`.